Makefile

```diff
@@ -58,32 +56,27 @@ copycomp:
 
 # Build api docs
 updateapispec:
-	cp $(K8SROOT)/api/openapi-spec/swagger.json gen-apidocs/generators/openapi-spec/swagger.json
+	cp $(K8SROOT)/api/openapi-spec/swagger.json gen-apidocs/config/swagger.json
 
 api: cleanapi
-	go run gen-apidocs/main.go --config-dir=gen-apidocs/generators --munge-groups=false
+	go run gen-apidocs/main.go --config-dir=gen-apidocs --munge-groups=false
 
-# NOTE: The following "sudo" may go away when we remove docker based api doc generator
 cleanapi:
-	sudo rm -rf $(shell pwd)/gen-apidocs/generators/build
-	sudo rm -rf $(shell pwd)/gen-apidocs/generators/includes
+	rm -rf $(shell pwd)/gen-apidocs/build
 
 copyapi:
+	mkdir -p $(APIDST)
 	cp $(APISRC)/build/index.html $(APIDST)/index.html
-	cp $(APISRC)/build/navData.js $(APIDST)/navData.js
-	cp $(APISRC)/static/scroll.js $(APIDST)/scroll.js
+	# copy scroll.js, jquery.scrollTo.min.js and the new navData.js
+	mkdir -p $(APIDST)/js
+	cp $(APISRC)/build/navData.js $(APIDST)/js/
+	cp $(APISRC)/static/js/* $(APIDST)/js/
+	# copy stylesheet.css, bootstrap.min.css, font-awesome.min.css
 	mkdir -p $(APIDST)/css
-	cp $(APISRC)/static/stylesheet.css $(APIDST)/css/stylesheet.css
-	cp $(APISRC)/static/bootstrap.min.css $(APIDST)/css/bootstrap.min.css
-	cp $(APISRC)/static/jquery.scrollTo.min.js $(APIDST)/jquery.scrollTo.min.js
-	cp $(APISRC)/static/font-awesome.min.css $(APIDST)/css/font-awesome.min.css
+	cp $(APISRC)/static/css/* $(APIDST)/css/font-awesome.min.css
```

Comment:
@tengqm , would you double check line 76. 
```
cp $(APISRC)/static/css/* $(APIDST)/css/
```

Reply:
Besides the few nits, looks fine 😃 . If time, would suggest updating the commit comment
with the updated directory structure.

Reply:
Done